tiledb/sm/filesystem/s3.cc

```diff
@@ -112,9 +136,18 @@ Status S3::init(
         Status::S3Error("Can't initialize with null thread pool."));
   }
 
+  options_.loggingOptions.logLevel =
+      aws_log_name_to_level(s3_config.logging_level_);
+
   // Initialize the library once per process.
   std::call_once(aws_lib_initialized, [this]() { Aws::InitAPI(options_); });
 
+  if (options_.loggingOptions.logLevel != Aws::Utils::Logging::LogLevel::Off) {
+    Aws::Utils::Logging::InitializeAWSLogging(
```

Comment:
Does this need to be within the `std::call_once` block? Or is it ok to happen multiple times?

Reply:
AFAICT it is safe, it just takes the value passed in the most recent constructor. I tested from python with multiple contexts
```
create ctx1: Trace
call w/ ctx1 --> log file
create ctx2: Off
call again with ctx1 -- no log file
```

I think that's probably fine because this is only going to be used for debugging.